docs/src/routes/docs/[...3]modules/common.md

```diff
@@ -0,0 +1,22 @@
+# Common
```

Comment:
Do we need this? No one should ever have to install this separately

Reply:
Good catch. We'll remove.